src/test/java/com/serenitydojo/fruitmarket/TheCatalog.java

```diff
@@ -10,49 +11,43 @@
 
 public class TheCatalog {
```

Comment:
by convention the test class should be called "CatalogTest".. or something else with test at the end